src/TimeGrid.jsx

```diff
@@ -77,9 +80,24 @@ export default class TimeGrid extends Component {
       this.measureGutter()
     }
 
+    if (this._scrollRatio) {
+      const { content } = this.refs;
+      content.scrollTop = content.scrollHeight * this._scrollRatio;
+      // Only do this once
+      this._scrollRatio = null;
```

Comment:
so the `calculateScroll` function is called from `componentDidMount` (which of course is only called once) and `componentDidUpdate` which gets called every time something is rendered, but is not shown on the diff, but [it's there](https://github.com/intljusticemission/react-big-calendar/pull/122/files#diff-fe4a2572cc6abfc6a7df5455b234527eL75).